src/icons/hicolor/scalable/apps/yast-migration.svg

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The `yast2-migration` package migrates the installed product to a newer service pack level, e.g. SLES12 SP1 to SLES12 SP2 (or newer if available).

For me the icon more suggest migrating (cloning) the installed system to somewhere else.

I think replacing the hard drive icon with a CD/DVD icon would be better. Or you could even use a package symbol as in reality the migration uses remote HTTP repositories, not physical CD/DVD media.

Reply:
Maybe something similar or even the same as in [yast-upgrade.svg](https://github.com/yast/yast-installation/blob/a51f73b1833fa3c65de7701800c87777558915ea/src/icons/hicolor/scalable/apps/yast-upgrade.svg) would be better.